<launch>
	<arg name="robot_name" default="robot1"/>
        <arg name="other_robot_name" default="robto2"/>

	<group ns="$(arg robot_name)">

		<include file="(find roscore_communication)/launch/includes/get_other_pose">
			<arg name="other_robot_name" value="$(arg other_robot_name)"/>
			<arg name="sub_topic" value="/$(arg other_robot_name)/robot_pose"/>
			<arg name="pub_topic" value="other_robot_pose/$(arg other_robot_name)_pose"/>
			<arg name="my_mergemap_frame" value="$(arg robot_name)/merge_map"/>
		</include>
		
	</group>
</launch>
